extension/src/test/fixtures/plotsDiff/output/multiSource.ts

```diff
@@ -0,0 +1,5 @@
+import { getMultiSourceOutput } from '..'
+
+const data = getMultiSourceOutput()
+
+export default data
```

Comment:
[F] This uses the regular pattern if we want to include the multi-source plots in a new story then I should be able to wire them up pretty quickly.